persistence/mongodb-record/src/main/scala/net/liftweb/mongodb/record/MongoMetaRecord.scala

```diff
@@ -280,6 +292,33 @@ trait MongoMetaRecord[BaseRecord <: MongoRecord[BaseRecord]]
   }
 
   /**
+    * replaces document with new one with given id. if `upsert` is set to true inserts new document
+    * in similar way as save() from sync api
+    */
+  def replaceOneAsync(inst: BaseRecord, upsert: Boolean, concern: WriteConcern) = {
```

Comment:
What do you think of adding default values for upsert (true) and concern (ACKNOWLEDGED)? 

Also, please add an explicit return type.